frontend/src/components/editCard/EditHeader.tsx

```diff
@@ -45,36 +45,40 @@ const EditHeader: FC<EditHeaderProps> = ({ edit }) => {
       case OperationEnum.MODIFY:
         return (
           <>
-            <Col xs={2} className="text-end">
-              Modifying {edit.target_type.toLowerCase()}:
+            <Col xs={2} className="fw-bold text-end">
+              Modifying {edit.target_type.toLowerCase()}
             </Col>
-            <Col>{renderTargetLink(edit.target)}</Col>
+            <Col className="pl-1">{renderTargetLink(edit.target)}</Col>
```

Comment:
`pl-1` is bootstrap 4, but it **doesn't look like** `ps-1` is needed here if your intention is to line it up with the fields below it.
```suggestion
            <Col>{renderTargetLink(edit.target)}</Col>
```

Reply:
Thanks, I was trying to line things up, but I (tried to) abandoned it.